pkg/common/utils/resource/pod.go

```diff
@@ -729,6 +752,46 @@ func getMultiSecretVolumeAndVolumeMount(bSpec *v1.BaseSpec, componentType v1.Com
 	return volumes, volumeMounts
 }
 
+func GetMultiSecretVolumeAndVolumeMountWithCommonSpec(cSpec *dv1.CommonSpec, componentType dv1.DisaggregatedComponentType) ([]corev1.Volume, []corev1.VolumeMount) {
+	var volumes []corev1.Volume
+	var volumeMounts []corev1.VolumeMount
+
+	defaultMountPath := ""
+	switch componentType {
+	case dv1.DisaggregatedFE, dv1.DisaggregatedBE, dv1.DisaggregatedMS:
+		defaultMountPath = secret_config_path
+	default:
+		klog.Infof("GetMultiSecretVolumeAndVolumeMountWithCommonSpec componentType %s not supported.", componentType)
+	}
+
+	for _, secret := range cSpec.Secrets {
+		path := secret.MountPath
+		if secret.MountPath == "" {
+			path = defaultMountPath
```

Comment:
path must be unique,  You need to add the unique verification of the mounting path. If you do not pass, print it in an wrong log. like this config:
```yaml
  feSpec:
    secrets:
    - secretName: secret1
    - secretName: secret2
```

Reply:
In the controller, it will be judged whether the path is unique or not.
<img width="745" alt="image" src="https://github.com/user-attachments/assets/d3548601-c1a1-4c7b-82c4-d3a746c09255" />